app/controllers/lcms/engine/api/resources_controller.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Lcms
+  module Engine
+    module Api
+      class ResourcesController < BaseController
+        def index
+          resources = Resource
```

Comment:
we can move `.all` here

Reply:
Why?

Reply:
There will be `.all` for `.where` too. Anyway up to you, it's mostly just preference just pass ready result to json serializer.